unit-tests/shared/src/test/scala/org/scalanative/testsuite/javalib/net/DatagramSocketTest.scala

```diff
@@ -0,0 +1,308 @@
+package org.scalanative.testsuite.javalib.net
+
+import java.net._
```

Comment:
The Scala Native project allows PR authors wide latitude when writing new code.   

This line is probably copied over from existing code.  
If true, that existing code is the way it is because it had
gone through a number of transformations since it was
first created.

I believe there is an opportunity for improvement here
but it is a matter of personal style and _not_ a requirement.

When writing code, __especially__ test code, I like to know
what is in the soup. That is, exactly what is being included
and from where.  Some may refer to the principle of least scope.  

Since `DatagramSocket` is being tested, `import java.net.DatagramSocket._` makes using the methods under test less tedious and closely models what users are
likely to do.

For other imports, I usually use practice recommended,
at least at one point, of explicitly importing <= 3 members from a class, and switching to _ (or * if writing Scala 3 code only).

Then again, for something like `SocketHelpers` I usually import just the class name and use that to qualify at the use site (as you did in one of the files).  The also works for
helping to distinguish a posixlib mumble.send() from a
javalib one.